src/pages/api/link-bot-community.ts

```diff
@@ -0,0 +1,55 @@
+import { API_URL } from '@/utils/config'
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { API_KEY } from '../../utils/config'
```

Comment:
You should import env variables directly from `process` rather than from a file that is shared between client and server. To make it clear **No environment variable should be instantiated on `utils/config.ts`**